src/main/resources/RiftSystem.yml

```diff
@@ -0,0 +1,5 @@
+allowedWorlds:
+ - "daily-world"
+ - "daily-world.*"
+
+portal_spawn_probability: 0.005
```

Comment:
In YML, there should be a new line to tell YML that is the EOF (End Of File)

Reply:
As far as I know, EOF is just standard EOF and there's no need to add a Line Break there...
I mean I did already test it and it's working completely fine.

Furthermore the YAML Specs don't require that at all anyway, so I'd be interested in where you got that from?